modules/olm-operator-lifecycle-manager.adoc

```diff
@@ -179,38 +181,421 @@ required to update a CSV to the latest version in a channel (stepping through
+ * `olm.opgroup.permissions/aggregate-to-edit: <operatorgroup_name>`
+
+|`<kind>.<group>-<version>-view`
+|Verbs on `<group>` `<kind>`:
+
+* `get`
+* `list`
+* `watch`
+
+Aggregation labels:
+
+* `rbac.authorization.k8s.io/aggregate-to-view: true`
+* `olm.opgroup.permissions/aggregate-to-view: <operatorgroup_name>`
+
+|===
+
+[[resources-additional-roles-rolebindings]]
+.Additional Roles and RoleBindings
+* If there is exactly one target namespace listed that contains `*`, then a
+ClusterRole and corresponding ClusterRoleBinding are generated for each
+permission defined in the CSV's permissions field. All resources generated are
+given the `olm.owner: <csv_name>` and `olm.owner.namespace: <csv_namespace>`
+labels.
+* If not, then all Roles and RoleBindings in the Operator namespace with the
+`olm.owner: <csv_name>` and `olm.owner.namespace: <csv_namespace>` labels are
+copied into the target namespace.
+
+[discrete]
+=== Copied CSVs
+
+OLM will create copies of all active member CSVs of an OperatorGroup in each of
+that OperatorGroup's target namespaces. The purpose of a copied CSV is to tell
+users of a target namespace that a specific Operator is configured to watch
+resources created there. Copied CSVs have a status reason `Copied` and are
+updated to match the status of their source CSV. The `olm.targetNamespaces`
+annotation is stripped from copied CSVs before they are created on the cluster.
+Omitting the target namespace selection avoids an unnecessary information leak.
+Copied CSVs are deleted when their source CSV no longer exists or the
+OperatorGroup that their source CSV belongs to no longer targets the copied
+CSV's namespace.
+
+[discrete]
+=== Static OperatorGroups
+
+An OperatorGroup is _static_ if its `spec.staticProvidedAPIs` field is set to
+`true`. As a result, OLM does not modify the OperatorGroup's `olm.providedAPIs`
+annotation, which means that it can be set in advance. This is useful when a
+user wants to use an OperatorGroup to prevent resource contention in a set of
+namespaces, but does not have active member CSVs that provide the APIs for those
+resources.
+
+Below is an example of an OperatorGroup that protects prometheus resources in
+all namespaces with the `something.cool.io/cluster-monitoring: "true"`
+annotation:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: cluster-monitoring
+  namespace: cluster-monitoring
+  annotations:
+    olm.providedAPIs: Alertmanager.v1.monitoring.coreos.com,Prometheus.v1.monitoring.coreos.com,PrometheusRule.v1.monitoring.coreos.com,ServiceMonitor.v1.monitoring.coreos.com
+spec:
+  staticProvidedAPIs: true
+  selector:
+    matchLabels:
+      something.cool.io/cluster-monitoring: "true"
+----
+
+[discrete]
+=== OperatorGroup intersection
+
+Two OperatorGroups are said to have _intersecting provided APIs_ if the
+intersection of their target namespace sets is not the empty set and the
```

Comment:
maybe s/the empty set /an empty set

I thought you could have only one OperatorGroup per namespace:
https://github.com/openshift/openshift-docs/pull/13907/files#diff-de2eac7c074474881ff71eeded85d7c7R237